external/src/main/resources/application.properties

```diff
@@ -9,23 +9,26 @@ kafka-streams.consumer.session.timeout.ms=250
 kafka-streams.consumer.heartbeat.interval.ms=200
 #external.org.hawkular.alerts.ui.path=/home/michael/projects/hawkular-alerts-ui
 
+# Kafka bootstrap applies to all topics
+kafka.bootstrap.servers=localhost:9092
+
 # Source <= hosts
 mp.messaging.incoming.kafka-hosts.connector=smallrye-kafka
-mp.messaging.incoming.kafka-hosts.topic=hosts
-mp.messaging.incoming.kafka-hosts.bootstrap.servers=localhost:9092
+mp.messaging.incoming.kafka-hosts.topic=platform.inventory.host-egress
+# mp.messaging.incoming.kafka-hosts.bootstrap.servers=localhost:9092
 mp.messaging.incoming.kafka-hosts.group.id=correlation
 mp.messaging.incoming.kafka-hosts.key.deserializer=org.apache.kafka.common.serialization.LongDeserializer
 mp.messaging.incoming.kafka-hosts.value.deserializer=io.vertx.kafka.client.serialization.JsonObjectDeserializer
 # Source <= hosts
 
 mp.messaging.outgoing.kafka-examples.connector=smallrye-kafka
-mp.messaging.outgoing.kafka-examples.topic=hosts
+mp.messaging.outgoing.kafka-examples.topic=platform.custom-policies.actions-email
```

Comment:
Were these decided? I thought the proposal was actions.email ?

Reply:
See the message on Slack, they preferred to keep 3 parts separated by "." and then changed the last . with a -